tests/test_text.py

```diff
@@ -162,8 +163,49 @@ def test_text_zfill(data, fletcher_variant):
     ser_fr = pd.Series(fr_array)
 
     result_pd = ser_pd.str.zfill(max_str_len + 1)
-    result_fr = ser_fr.fr_text.zfill(max_str_len + 1)
+    result_fr = ser_fr.fr_strx.zfill(max_str_len + 1)
     result_fr = result_fr.astype(object)
     # Pandas returns np.nan for NA values in cat, keep this in line
     result_fr[result_fr.isna()] = np.nan
     tm.assert_series_equal(result_fr, result_pd)
+
+
+def test_fr_str_accessor(fletcher_array):
+    data = ["a", "b"]
+    ser_pd = pd.Series(data)
+
+    pd_stringmethods = getmodule(pd.core.strings.StringMethods).__name__  # type: ignore
```

Comment:
I would rather not test with `getmodule(…).__name__` as I could image that in future the implementation looks different. Tests should only check whether we do the right thing, not check that we use a specific approach to solve the problem.